draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -540,7 +540,7 @@ can be registered with IANA following the rules in {{IANA-Extra-Cause}}.
 
 If list of protection engines contained in the INIT signal doesn't
 contain at least an entry that fits the list of protection engines at
-the responder, the responder will reply with an ERROR chunk with error
+the responder, the responder will reply with an ABORT chunk with error
```

Comment:
So is this really an ABORT? I think it might be an error and the INIT-ACK will simply include the error saying, although I support the Crypto chunk we have no common engines.